app/controllers/journals_controller.rb

```diff
@@ -2,6 +2,7 @@ class JournalsController < ApplicationController
   before_action :find_user, only: %i[new create index show destroy update]
   before_action :find_journal, only: %i[show destroy update find_journal]
   before_action :require_login
```

Comment:
Maybe we can move the `before_action :require_login` to the application_controller ?

Reply:
Could you please explain what happens in that case?

Reply:
It does not get called in the same Situations. Laila said I should leave it like this. But thanks for the hint